server/src/linker.js

```diff
@@ -42,3 +42,13 @@ exports.imageLink = function (urlBase, resource) {
   }
   return urlBase + "/images" + resource;
 };
+
+exports.contentLink = function (urlBase, resource) {
+  if (! resource.startsWith("/")) {
+    resource = "/" + resource;
+  }
+  if (resource.startsWith("/content")) {
+    throw new Error("imageLink URL should not start with /content: " + resource);
+  }
+  return urlBase + "/content" + resource;
+};
```

Comment:
Maybe?  With this function I'd worry about passing in the wrong urlBase in some place, while the Shot if instantiated correctly should be okay.  Also I suppose the Shot could take the content base URL directly from configuration?  Since it's not dynamically determined from the request at all.